src/helper.rs

```diff
@@ -92,167 +92,27 @@ mod filter {
 
 /// Render html to command-line
 mod html {
-    // use crate::Error;
-    use colored::{Color, Colorize};
-    use escaper::decode_html;
-    use regex::Regex;
-    pub enum Token {
-        Plain(String),
-        Bold(String),
-        Sup(String),
-        Sub(String),
-        Font((String, Color)),
-        Eof(String),
-    }
-
+    use scraper::Html;
     /// Html render plugin
     pub trait HTML {
-        fn ser(&self) -> Vec<Token>;
         fn render(&self) -> String;
     }
 
-    pub fn superscript(n: u8) -> String {
-        match n {
-            x if x >= 10 => format!("{}{}", superscript(n / 10), superscript(n % 10)),
-            0 => "⁰".to_string(),
-            1 => "¹".to_string(),
-            2 => "²".to_string(),
-            3 => "³".to_string(),
-            4 => "⁴".to_string(),
-            5 => "⁵".to_string(),
-            6 => "⁶".to_string(),
-            7 => "⁷".to_string(),
-            8 => "⁸".to_string(),
-            9 => "⁹".to_string(),
-            _ => n.to_string(),
-        }
-    }
```

Comment:
plz check if scrapper can support this

Reply:
however, I do not think is nessesary, and this relpace use `^` and `_`, if you think this must be use the utf-8 character, I can do some manual work to recover this. :)

Reply:
these optimizations are from https://github.com/clearloop/leetcode-cli/pull/12 and https://github.com/clearloop/leetcode-cli/pull/39, we better to keep these for respecting the previous contributors, btw, I think there is a escraper in our crate also lol https://github.com/clearloop/leetcode-cli/pull/11/files